src/api.js

```diff
@@ -5,16 +5,25 @@ import { loadMovie } from "./movies.js";
 import { getReviewsForMovieWithId } from "./reviewList.js";
 import loadScreenings from "./loadScreenings.js";
 import { filterScreenings } from "./filterScreenings.js";
+import { login, isAuthenticated } from "./auth.js";
 
 const apiRouter = express.Router();
 
+apiRouter.post("/token", (req, res) => { 
+	login(req, res);
+})
+
 apiRouter.get("/movies/:id", (req, res) => {
 	res.send(`movie with id of ${req.params.id}`);
 });
 
-apiRouter.post("/movies/:movieId/reviews", express.json(), async (req, res) => {
+apiRouter.post("/movies/:movieId/reviews", isAuthenticated, express.json(), async (req, res) => {
+	if (req.params.movieId === "TEST") {
```

Comment:
What is the purpose of this? It's not usually desirable to have code paths that are only relevant for tests (because they what are you really testing?).

If the purpose is to be able to test the `isAuthenticated` middleware, it's possible to write tests for that middleware without using a real route, and then trust that `express` will properly use the middleware.

If the purpose is to test in an integration test that your route properly checks for authentication, it would be better to use the real code path (i.e. no special if statement for tests) but mock out the part that actually posts the review to the CMS.

Finally, if you the purpose of this is to avoid posting the review, it's not actually doing that, because even if `res.status().json()` happens on the next line, that doesn't mean the function stops executing. Only the `return` statement (or an exception) can do that. So even if this if statement is entered, the code will still also execute `postReview()` (which will probably fail).